### PR TITLE
Allow pausing all running video players

### PR DESCRIPTION
**Pull request by pzotov**
Task https://trello.com/c/w12OFjjg/25411-stop-media-of-lr-after-opening-another-lr-from-map

### CODE REVIEW

File: lib/material/internal/material_player.flow [hunk 1]
```diff
@@ -19,8 +19,15 @@ MVideoPlayer2T(manager : MaterialManager, parent : MFocusGroup, m : MVideoPlayer
 		getExternalPlayerURL(m.filename),
 		\url -> {
 			enableYoutubeJsApi = isSome(getYouTubeURL(m.filename)) && containsStruct(m.style, MPlayerYoutubeJsApi());
-			if (enableYoutubeJsApi) m2t(MVideoYoutubePlayer(m.filename, m.wh, m.style), p)
-			else MVideoPlayer2MRealHTML(url, p, m, m2t)
+			playing = extractStruct(m.style, FVideoPlay(make(false))).play;
+			TConstruct(
+				[
+					makeSubscribe2(manager.videoRunning, \videoRunning -> if (!videoRunning) next(playing, false)),
```
Comment (NikitaFil) [inline lib/material/internal/material_player.flow:25]: These two lines are fBidirectionalLink

File: lib/material/internal/material_player.flow [hunk 2]
```diff
@@ -793,6 +800,8 @@ MVideoPlayer2T(manager : MaterialManager, parent : MFocusGroup, m : MVideoPlayer
 				TConstruct(
 					concat(
 						[
+							makeSubscribe2(manager.videoRunning, \videoRunning -> if (!videoRunning) next(playing, false)),
```
Comment (NikitaFil) [inline lib/material/internal/material_player.flow:803]: Same

File: lib/material/internal/material_player.flow (hunk 1 above)
Comment (pzotov) [reply lib/material/internal/material_player.flow]: updated

**Review by NikitaFil — APPROVED** [review]
LGTM

**Pull request merged**